src/Controller/UsersController.ts

```diff
@@ -84,4 +86,13 @@ export class UsersController extends BaseHttpController {
 
     return this.json(result, 400)
   }
+
+  @httpGet('/:userUuid/params')
+  async keyParams(request: Request): Promise<results.JsonResult> {
+    const result = await this.getUserKeyParams.execute({
+      email: <string> request.query.email
```

Comment:
cast to string because we trust that the internal requests will always contain email as string?

Reply:
Yeah, my bad - will turn into an optional param

Reply:
Actually it makes more sense to get it by `userUuid` from the request query ... refactored the use case a bit to accept that workflow.